packages/stream-object/src/index.ts

```diff
@@ -54,9 +54,8 @@ export const streamObject = async <T extends Schema>(options: StreamObjectOption
 
     return {
       chunkStream,
-      finishReason,
       partialObjectStream,
+      stepStream,
       textStream,
-      usage,
```

Comment:
Is the removal of `usage` here intended?

Reply:
> Is the removal of `usage` here intended?

yes